### ISSUE TITLE
fsharpi crashes on El Capitan

### ISSUE THREAD

**Issue by naveensrinivasan** [fsharp/fsharp#502]
The fsharpi crashes on my mac  within Xamarin Studio
```
> /Library/Frameworks/Mono.framework/Versions/4.0.4/bin/fsharpi: line 25: exec: mono: not found

Session termination detected. Press Enter to restart.

/Library/Frameworks/Mono.framework/Versions/4.0.4/bin/fsharpi: line 25: exec: mono: not found

Session termination detected. Press Enter to restart.

```

Here is the content of fsharpi
```
#!/bin/sh
EXEC="exec "
mono="/usr/local/bin/mono"

if test x"$1" = x--debug; then
   DEBUG=--debug
   shift
fi

if test x"$1" = x--gdb; then
   shift
   EXEC="gdb --eval-command=run --args "
fi

if test x"$1" = x--valgrind; then
  shift
  EXEC="valgrind $VALGRIND_OPTIONS"   
fi

# Beware this line must match the regular expression " (\/.*)\/fsi\.exe" when fsi.exe is fsi.exe.
# That's because the FSharp MonoDevelop addin looks inside the text of this script to determine the installation
# location of the default FSharp install in order to find the FSharp compiler binaries (see 
# fsharpbinding/MonoDevelop.FSharpBinding/Services/CompilerLocationUtils.fs). That's a pretty unfortunate
# way of finding those binaries. And really should be changed.
$EXEC mono $DEBUG $MONO_OPTIONS /Library/Frameworks/Mono.framework/Versions/4.0.4/lib/mono/4.5/fsi.exe --exename:$(basename "$0") "$@"
```

It works perfectly fine on the terminal.

Any clues as to what could be wrong?

```
  /  uname -a
Darwin Naveens-MBP.home 15.0.0 Darwin Kernel Version 15.0.0: Wed Aug 26 16:57:32 PDT 2015; root:xnu-3247.1.106~1/RELEASE_X86_64 x86_64

```

**Comment by adamchester** [comment]
Same issue here. Appears to be related to recent El Capitan changes.

When I start `XamarinStudio` from the terminal (with `fsharpc` and `fsharpi` on the `PATH`), the F# interactive works.

**Comment by naveensrinivasan** [comment]
Thanks  :+1:  That worked! I am trying to look for a more permanent solution.

**Comment by dsyme** [comment]
Any idea what changes we need to make?

**Comment by naveensrinivasan** [comment]
This is not an issue any more. It has been fixed in the mono 4.2.1
```
$EXEC /Library/Frameworks/Mono.framework/Versions/4.2.1/bin/mono $DEBUG $MONO_OPTIONS /Library/Frameworks/Mono.framework/Versions/4.2.1/lib/mono/4.5/fsi.exe --exename:$(basename "$0") "$@"

```

**Comment by dsyme** [comment]
Thanks for the update!
